proto/api/v1/config_service.proto

```diff
@@ -99,4 +103,8 @@ service ConfigService {
       body: "*"
     };
   }
+
```

Comment:
Can you add a one liner comment here saying what this endpoint returns and then regenerate proto sources? Comments here make it to the API docs.